src/main/python/interpolate.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+'''
+Anserini: A Lucene toolkit for replicable information retrieval research
```

Comment:
mangled boilerplate... use https://github.com/castorini/anserini/blob/master/src/main/python/run_es_regression.py#L1

Reply:
Interesting... if we just copy and paste, the line breaks disappear. Anyway, fixed!